domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -1542,6 +1543,11 @@ private void populateMetaData(Object entity, Map<String, String> okapiHeaders, S
     //the created date and by fields are stored in the db in separate columns on insert trigger so that even if
     //we overwrite them here, the correct value will be set in the db level via a trigger on update
     String json;
+    Metadata md = new Metadata();
+    md.setUpdatedDate(new Date());
+    md.setCreatedDate(new Date());
+    md.setCreatedByUserId(SYSTEM_USER_ID);
+    md.setUpdatedByUserId(SYSTEM_USER_ID);
```

Comment:
Should we fill updatedByUsername and createdByUsername fields?

Reply:
The case is, that we do not have information about user name from the request.
Also there is quite stage behavior of data base trigger for record with metadata ([trigger](https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/main/resources/templates/db_scripts/metadata.ftl)).
It erases createdByUsername and updatedByUsername on update